webapp/common.js

```diff
@@ -40,20 +40,18 @@ function setWindow(window, back) {
           .click(() => {clearStatus(); setWindow(back); return false;})
           .removeAttr('href')
           .removeClass('button-hidden');
-    } else {
-        backButton.attr('href', MESSAGES['issuers-overview-page']);
-        if (isInApp)
-            backButton.addClass('button-hidden');
+    } else if (history.length == 1) {
+        backButton.addClass('button-hidden');
```

Comment:
You need to add the `href` if `history.length > 1` here and set
`backButton.attr('href', 'history.back()')`

Otherwise, the back button does not do anything if I'm right.